### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by sandrotosi** [matplotlib/basemap#405]
GH report 48 commits since 1.1.0 was tagged, maybe it's time for a new release of basemap?

**Comment by megies** [comment]
Totally +1 especially since 1.1.0 will break with matplotlib 2.3, see #382.

**Comment by fjansson** [comment]
On this note, would it be possible to get a new release in Pypi also? The version there is still 1.0.7.
I just hit matplotlib issue https://github.com/matplotlib/basemap/issues/395 - basemap 1.0.7 apparently does not work with matplotlib 2.2.2, both of which I installed with pip.

I see this has been discussed elsewhere too, though: https://github.com/matplotlib/basemap/issues/251

**Comment by hany** [comment]
Hi there,

Just ran into #395 as well. Wondering when a new version will be pushed to PyPI?

**Comment by gwerbin** [comment]
Installing basemap 1.2 with Conda actually downgraded my Matplotlib all the way to 1.5. Yikes.

**Comment by gwerbin** [comment]
No. I think my setup was just mildly "broken", I have a ton of packages installed and I'm deep in dependency hell anyway.

**Comment by xylar** [comment]
With the merge of #454, it would be especially nice to have a release of basemap that supports the latest proj4/pyproj.  basemap is starting to cause major compatibility problems with other packages on conda-forge, which have moved on to pyproj>=2 and proj4>=6.

**Comment by molinav** [comment]
There is a new release available (1.3.0). This release also brings some precompiled binary wheels to PyPI, and `basemap` is split in three parts in a similar way to what Anaconda does. With `pip`, now it should be as simple as:
```sh
python -m pip install basemap
```
and, in case that you also need the high-resolution datasets:
```sh
python -m pip install basemap-data-hires
```
